styles.scss

```diff
@@ -248,6 +248,40 @@ $color-link-bold-dark: lighten($color-link-bold, 20%);
   }
 }
 
+// Explicit dark mode via CSS classes
+html.dark,
+body.dark,
+html[data-theme="dark"],
+body[data-theme="dark"],
```

Comment:
```suggestion
```

This was repeated I believe

Reply:
Oh, the 2nd one needs to be `body[data-bs-theme="dark"] `